rustv1/examples/dynamodb/README.md

```diff
@@ -62,6 +63,18 @@ functions within the same service.
 
 
 
+#### Connect to a local instance
+
+This example shows you how to Override the endpoint URL to connect to a local development deployment of DynamoDB and an AWS SDK.
```

Comment:
Replace with "This example shows you how to override the endpoint URL to connect to a local development deployment of DynamoDB using an AWS SDK."

Reply:
Done